pages/common/scheme.md

```diff
@@ -0,0 +1,23 @@
+# scheme
+
+> MIT Scheme language interperter and REPL (interactive shell)
+
+- Open an interactive shell (REPL):
+
+`scheme`
+
+- Run a scheme program (with no REPL output):
+
+`scheme --quiet < script.scm`
+
+- Load a scheme program into the REPL:
+
+`scheme --load {{script.scm}}`
+
+- Load scheme expressions into the REPL:
+
+`scheme --eval "(define foo 'x)"`
```

Comment:
expression in tokens too

Reply:
Should that be `{{"(define foo 'x)"}}` or `"{{(define foo 'x)}}"` ?